out/models.go

```diff
@@ -10,11 +10,12 @@ type Request struct {
 }
 
 type Params struct {
-	From        string `json:"from"`
-	File        string `json:"file"`
-	To          string `json:"to"`
-	Acl         string `json:"acl"`
-	ContentType string `json:"content_type"`
+	From             string `json:"from"`
+	File             string `json:"file"`
+	To               string `json:"to"`
+	Acl              string `json:"acl"`
+	ContentType      string `json:"content_type"`
+	DisableMultiPart bool   `json:"disable_multipart"`
```

Comment:
Should this be in `s3resource.Source` instead, since it's unlikely to change from `put` to `put`? Either the endpoint supports multipart uploads or it doesn't.